docs/tutorials/geomesa-quickstart-fsds.rst

```diff
@@ -171,12 +171,62 @@ contained in ``org.geomesa.example.data.GDELTData``:
 -  ``getTestData`` parses an embedded TSV file to create ``SimpleFeature`` objects
 -  ``getTestQueries`` illustrates several different query types, using CQL (GeoTools' Contextual Query Language)
 
-Visualize Data With GeoServer (optional)
-----------------------------------------
+Visualize Data (optional)
+-------------------------
+
+There are two options to visual the data ingested by this quick start. The easiest option is to use the
+``export`` command of the GeoMesa FSDS tools distribution. For a more production ready example, you can
+alternatively stand up a GeoServer and connect it to your FSDS instance.
+
+Visualize Data With Leaflet
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+    In order to successfully run this command you must have a computer that is connected to the internet
+    in order to access external leaflet resources.
```

Comment:
minor wording - could we change the wording here to get rid of the first 'in order'.